dMasterServer/MasterServer.cpp

```diff
@@ -176,12 +176,16 @@ int main(int argc, char** argv) {
 	}
 
 	// Run migrations should any need to be run.
-	MigrationRunner::RunSQLiteMigrations();
+	MigrationRunner::RunSQLiteMigrations();	
 
 	//If the first command line argument is -a or --account then make the user
 	//input a username and password, with the password being hidden.
-	if (argc > 1 &&
-		(strcmp(argv[1], "-a") == 0 || strcmp(argv[1], "--account") == 0)) {
+	bool doesAnyAccountExist = Database::Get()->GetAccountCount() > 0;
+	if (!doesAnyAccountExist) {
+		LOG("No accounts exist in the database.  Please create an account.");
+	}
+	if ((argc > 1 &&
+		(strcmp(argv[1], "-a") == 0 || strcmp(argv[1], "--account") == 0)) || !doesAnyAccountExist) {
```

Comment:
As much as I'd like this for users, dropping into interactive mode with an empty DB may break the initial docker startup (e.g. if in detached mode)

Maybe add an env var there that can prevent this?

Reply:
ill add a define to set this to false for docker and for any other automated setups like docker

Reply:
Please check this code again, it has been updated